src/services/storages.ts

```diff
@@ -69,20 +31,26 @@ export class SimpleInMemoryStorageService implements StorageService {
   /**
    * @inheritdoc
    */
-  public set<T>(name: string, value: T, options: ValueOptions): Promise<T> {
+  public set<T>(name: string, value: T, options?: ValueOptions): Promise<T> {
```

Comment:
Why need values in storage without TTL?

Reply:
@AlekNS We need to store consumer's Authenticator private key permanently until consumer decides to disable 2FA. So I had to refactor `set` method.

Reply:
Hm... is it of "verification" responsibility to store this persistent data?

Reply:
@AlekNS This secret is required to `verify` 2FA code which is definitely `backend-verification` responsibility. What service should store it if not `verification`?

Reply:
In my opinion, it's verification service's responsibility so it's OK